Lib/ftplib.py

```diff
@@ -713,28 +713,13 @@ class FTP_TLS(FTP):
         '221 Goodbye.'
         >>>
         '''
-        ssl_version = ssl.PROTOCOL_TLS_CLIENT
```

Comment:
Why is it removed?

Reply:
If you provide a SSLContext, you get different protocol than ``FTP_TLS.ssl_version``. This removal is not required, it's more to make the API consistent with the other modified modules: the SSLContext constructor is the only place setting default parameter values. Do you prefer to keep it?

Reply:
All protocols except PROTOCOL_TLS, TLS_CLIENT, and TLS_SERVER are deprecated anyways because OpenSSL is going to remove them eventually.

Reply:
@tiran: Do you see a reason to keep ssl_version to specify a different "ssl version" when context parameter is omitted?

The practical problem is more that applications relying on this feature may not notice that overriden ssl_version are now ignored silently.

Note: I hate mentions of "ssl", since SSL no longer exists, it was replaced with TLS, but the Python module is still called "ssl"... as OpenSSL ;-)

Reply:
There is no reason to keep the variable here. ``ssl.PROTOCOL_TLS_CLIENT`` is the only correct and fully supported value. In 3.13 I will remove all the other protocols. Only ``ssl.PROTOCOL_TLS_CLIENT`` and ``ssl.PROTOCOL_TLS_SERVER`` will stay.